server/main.py

```diff
@@ -98,8 +98,8 @@ async def index(request):
 
     # Prepare the job's configuration and write it to a blob.
     config = get_baseline_config(server_config, dataset, access_level, output_prefix)
-    if 'config' in params:  # Update with user-specified configs.
-        update_dict(config, params['config'])
+    if user_config := params.get('config'):  # Update with user-specified configs.
```

Comment:
Does this need to be fixed on the Cromwell side as well?

Reply:
Ah yes I'd assume so, to be honest I didn't even realise Cromwell used the config, but makes sense. Will send a PR